### DISCUSSION TITLE
One letter move by bot

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#14] [Q&A] [answered]
I find an one letter bot move to be unexpected. 

![Screen Shot 2023-04-15 at 10 35 31 AM](https://user-images.githubusercontent.com/4314771/232245468-76468a31-6c41-4ee8-869a-ff3ab524cd06.png)
logs:
[eagerwords.com-1681580287288.log](https://github.com/azadbolour/eagerwords-community/files/11240273/eagerwords.com-1681580287288.log)

[eagerwords.com-1681580232515.log](https://github.com/azadbolour/eagerwords-community/files/11240275/eagerwords.com-1681580232515.log)

[eagerwords.com-1681580212739.log](https://github.com/azadbolour/eagerwords-community/files/11240289/eagerwords.com-1681580212739.log)

**Comment by azadbolour** [comment, marked as answer]
Interesting. According to Merriam-Webster, 'S' is indeed an English word.

https://www.merriam-webster.com/dictionary/s

One letter words are available where they make sense, like in the first
play in games. Also, in the browser version, the user can configure the
"word connectivity" parameter to be optional. And in that case, one letter
plays can be made at any time. The problem with one-letter plays, of
course, is that they are highly vulnerable to takeover by the opponent.

Azad

**Comment by danconde** [comment]
Other one letter moves seem to be allowed.  I had thought 'M' was not allowed but the bot had made a move just prior, and resulted in a disconnected move.  But back to topic of one letter words, I had understood that only 'I' and 'a' are one letter words, with perhaps 'O' from poems.

A – an indefinite article
I – the first person singular nominative pronoun
O – an interjection, used in the translation of the vocative case of Latin, commonly used in poetry
